### PR TITLE
Audit questions BugBash fixes

### PR DESCRIPTION
**Pull request by alxnddr**
Bug bash feedback on audit questions:
1) add a unit to new columns
2) add hover state for column headers
3) do not show loader on audit table when sorting and filtering

How to test:
- Make sure you run Metabase EE
- Visit [audit questions page](http://localhost:3000/admin/audit/questions/all)
- Ensure hover state exists, columns have unit

Screen Shot 2021-06-11 at 01 15 42

### CODE REVIEW

File: enterprise/backend/src/metabase_enterprise/audit/pages/queries.clj [hunk 1]
```diff
@@ -95,9 +95,9 @@
                [:user_name       {:display_name "Created By",      :base_type :type/Text,    :remapped_from :user_id}]
                [:public_link     {:display_name "Public Link",     :base_type :type/URL}]
                [:cache_ttl       {:display_name "Cache Duration",  :base_type :type/Number}]
-               [:avg_exec_time   {:display_name "Average Runtime", :base_type :type/Integer}]
-               [:total_runtime   {:display_name "Total Runtime",   :base_type :type/Number}]
-               [:query_runs      {:display_name "Query Runs",      :base_type :type/Integer}]
+               [:avg_exec_time   {:display_name "Average Runtime (ms)", :base_type :type/Integer}]
+               [:total_runtime   {:display_name "Total Runtime (ms)",   :base_type :type/Number}]
+               [:query_runs      {:display_name "Query Runs (ms)",      :base_type :type/Integer}]
```
Comment (kulyk) [inline enterprise/backend/src/metabase_enterprise/audit/pages/queries.clj:100]: Now you can tell people you write in LISP :D

**Review by kulyk — APPROVED** [review]
Looks good!

File: enterprise/backend/src/metabase_enterprise/audit/pages/queries.clj (hunk 1 above)
Comment (alxnddr) [reply enterprise/backend/src/metabase_enterprise/audit/pages/queries.clj:100]: Haha it's time to add Clojure to my skills on LinkedIn 😄

**Comment by alxnddr** [comment]
@metabase/frontend-developers could someone please review the latest changes?

It hides loader when filtering and sorting audit table. Make sure you run Metabase EE.

Here is a short demo:
https://user-images.githubusercontent.com/14301985/122429398-6cf7ed00-cf9b-11eb-94a2-b9636e518e0c.mov

[Audit questions page](http://localhost:3000/admin/audit/questions/all)

**Review by kdoh — APPROVED** [review]
Looks great

**Pull request merged**